src/components/HeaderNavigation.vue

```diff
@@ -117,14 +118,12 @@ export default {
 		 * so we generate a new valid route object, get the final url back
 		 * decode it and use it as a direct string, which vue-router
 		 * does not encode afterwards
-		 *
-		 * @return {string|object}
 		 */
-		to() {
+		to(): string|object {
```

Comment:
```suggestion
		to(): string|Location {
```

(location from vue router no?)